examples/QuickJournal/QuickJournal/ViewModels/JournalEntryDetailsViewModel.cs

```diff
@@ -1,36 +1,52 @@
-﻿using System.Windows.Input;
+﻿using System;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using QuickJournal.Models;
+using QuickJournal.Services;
 using Realms;
 using Xamarin.Forms;
 
-namespace QuickJournal
+namespace QuickJournal.ViewModels
 {
-    public class JournalEntryDetailsViewModel
+    public class JournalEntryDetailsViewModel : BaseViewModel
     {
-        private Transaction _transaction;
+        private Transaction transaction;
```

Comment:
Nit pick here: I'd recommend sorting fields the same way you sort functions: public first, then private.